### PR TITLE
Add config file imports

### PR DESCRIPTION
**Pull request by ehankland**

### CODE REVIEW

File: perfkitbenchmarker/configs/__init__.py [hunk 1]
```diff
@@ -90,10 +93,44 @@
...
+      data = self.files[self.current_file_index].read(length)
+    return data
+
+
+def _GetImportFiles(config_file):
+  """Get a list of file names that get imported from config_file."""
+  config_path = data.ResourcePath(config_file)
+  with open(config_path) as f:
+    line = f.readline()
+    match = IMPORT_REGEX.match(line)
```
Comment (yuyantingzero) [inline perfkitbenchmarker/configs/__init__.py]: reuse regex_util?

File: perfkitbenchmarker/configs/__init__.py [hunk 2]
```diff
@@ -90,10 +93,44 @@
     'default.vm_count=4).')
 
 
+class _ConcatenatedFiles(object):
+
```
Comment (yuyantingzero) [inline perfkitbenchmarker/configs/__init__.py]: add comments for attributes

Comment (ehankland) [inline perfkitbenchmarker/configs/__init__.py]: done

File: perfkitbenchmarker/configs/__init__.py [hunk 3]
```diff
@@ -90,10 +93,44 @@
...
+  def read(self, length):
+    data = self.files[self.current_file_index].read(length)
+    while (not data) and (self.current_file_index + 1 < len(self.files)):
+      self.current_file_index += 1
+      data = self.files[self.current_file_index].read(length)
+    return data
+
+
+def _GetImportFiles(config_file):
+  """Get a list of file names that get imported from config_file."""
```
Comment (ehankland) [inline perfkitbenchmarker/configs/__init__.py]: done

File: perfkitbenchmarker/configs/__init__.py (hunk 1 above)
Comment (ehankland) [inline perfkitbenchmarker/configs/__init__.py]: regex util throws an exception if there's no match which is why i didnt use it

File: perfkitbenchmarker/configs/__init__.py [hunk 4]
```diff
@@ -90,10 +93,44 @@
...
+  config_path = data.ResourcePath(config_file)
+  with open(config_path) as f:
+    line = f.readline()
+    match = IMPORT_REGEX.match(line)
+    import_files = []
+    while match:
+      import_file = data.ResourcePath(match.group(1))
+      for file_name in _GetImportFiles(import_file):
```
Comment (ehankland) [inline perfkitbenchmarker/configs/__init__.py]: fixed

**Comment by ehankland** [comment]
PTAL

**Pull request merged**